modules/templates/helper.go

```diff
@@ -285,6 +285,13 @@ func NewFuncMap() []template.FuncMap {
 			}
 			return false
 		},
+		"octicon": func(icon string, mega bool) template.HTML {
```

Comment:
Could the `mega` argument be made a `interface` enabling a even shorter `{{octicon "lock"}}` for non-mega ones?

Reply:
Maybe accept a string `type` as the second argument so a mega one is `{{octicon "lock" "mega"}}` and regular one `{{octicon "lock"}}`. I find that much clearer then having nondescript booleans in the template.

Reply:
afaik that doesn't work, you need to supply all parameters to a template function.

Reply:
I could, however, add a second template helper for e.g. `{{mega-octicon "lock"}}`

Reply:
yeah that would be a acceptable alternative. Strange that you cannot use `interface{}` in template helpers thought.

Reply:
How about just taking two strings and using a abstract `icon` helper?

```
{{icon "octicon" "lock"}}
{{icon "mega-octicon" "lock"}}
{{icon "font-awesome" "lock"}}

Reply:
That would still need a way to distinguish the symbol name, since `mega-octicon-lock` doesn't exist.

Reply:
Maybe get rid of `mega` altogether and size them in CSS? We could even simplify it to single string `octicons-lock` which would map directly to the `id` in the spritesheet.

Reply:
To an extent we need a way to differentiate between normal and mega, because currently `mega-octicon` is how they are sized.  
Going through each template to figure out a CSS selector sounds rough.

Reply:
Yeah, that's why I suggested a `size` argument, to save oneself from going insane with CSS, e.g. `{{svg "octicons-lock" 30}}` would render `<svg width="30" height="30">...`. I think this way it's more universally usable, also for future libraries.

Reply:
And then the rest have a generic `svg` selector?
```css
display: inline-block;
vertical-align: text-top; /** middle? **/
fill: currentColor;
```

Reply:
Yes, generic low-level svg styles without dimensions. Here's some base styles that blueprint.js has for example:

````
svg {
  display: inline-block;
  fill: currentColor;
  flex: 0 0 auto;
  vertical-align: text-bottom; 
}
````

I guess `flex` is so it does not stretch in a flexbox context. Not sure about `vertical-align`, maybe keep current value.

Reply:
I got the current rules from the [octicons repo](https://github.com/primer/octicons/blob/master/lib/octicons_node/index.scss)

Reply:
Here's my [reference](https://github.com/palantir/blueprint/blob/06a186c90758bbdca604ed6d7bf639c3d05b1fa0/packages/core/src/components/icon/_icon.scss). I guess go with `text-top` if it looks okay.

Reply:
One more benefit of having size in HTML is that it allows the browser to "reserve" the area while painting even when CSS has not loaded yet which prevents elements from jumping on the page during rendering. Probably not that big of an issue currently because our CSS is render-blocking, but it may come up with lazy-loaded content.

Reply:
Alright, this is done.  
However, I chose the class `.svg` instead of the element `svg` and instead of `.icon` because `.icon` is used by fomantic and I didn't want to play mix-n-match with it any more than necessary.